From 249b11ccbf232c005a0784552e1f8292e92e8e19 Mon Sep 17 00:00:00 2001
From: ouni666 <fangwenbin@vip.qq.com>
Date: Thu, 19 Mar 2026 22:17:14 +0800
Subject: [PATCH] replace static lib with tutorial link target

Fix osd_tutorial_0 to use OSD_LINK_TARGET instead of hardcoded static library.

This allows successful linking in both shared and static builds.

---
 tutorials/osd/tutorial_0/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tutorials/osd/tutorial_0/CMakeLists.txt b/tutorials/osd/tutorial_0/CMakeLists.txt
index 57acb9c1..2e286ec0 100644
--- a/tutorials/osd/tutorial_0/CMakeLists.txt
+++ b/tutorials/osd/tutorial_0/CMakeLists.txt
@@ -15,7 +15,7 @@ osd_add_executable(osd_tutorial_0 "tutorials/osd"
 
 
 target_link_libraries(osd_tutorial_0
-    osd_static_cpu
+    ${OSD_LINK_TARGET}
 )
 
 install(TARGETS osd_tutorial_0 DESTINATION "${CMAKE_BINDIR_BASE}/tutorials")
-- 
2.52.0

